### DISCUSSION TITLE
Missing columns in Wiki?

### DISCUSSION THREAD

**Discussion by ollypsilon** [joyfullservice/msaccess-vcs-addin#370] [Q&A] [answered]
In Wiki paragraph on [supported objects](https://github.com/joyfullservice/msaccess-vcs-integration/wiki/Supported-Objects#detailed-list) is mentioned `The Testing column indicates whether a test item and testing code has been created in the Testing.accdb database to verify that particular item after import from source code. Test location indicates where to find the object in the database.`
But I can't find column "Testing" or "Test location".
![image](https://user-images.githubusercontent.com/52780811/202672393-4a7fbdab-4ed5-4e7a-9f02-d62d2770c641.png)

**Comment by joyfullservice** [comment, marked as answer]
@ollypsilon - Thanks for pointing this out! When I originally created this table in the Wiki a couple years ago, GitHub didn't work well with wider tables, so I decided to drop the extra columns. It seems that this limitation may have changed since then, as I can display the additional columns without a problem now. I went ahead and added the missing columns, and will try to fill out the data when I have a chance.  😄
